packages/itwin/tree-widget/src/e2e-tests/TreeWidget.test.ts

```diff
@@ -123,4 +119,33 @@ test.describe("should match image snapshot", () => {
     await treeWidget.locator(".components-activehighlight").waitFor();
     await takeScreenshot(page, treeWidget);
   });
+
+  test("button dropdown", async ({ page }) => {
+    await treeWidget.getByTitle("Search for something").click();
+    await treeWidget.getByTitle("More").click();
+    await page.locator(".tree-header-button-dropdown-container").waitFor();
+    await takeScreenshot(page, treeWidget);
+  });
+
+  test("tree selector", async ({ page }) => {
+    await treeWidget.getByRole("combobox").click();
+    await page.getByRole("listbox").waitFor();
+    await takeScreenshot(page, treeWidget);
+  });
+};
+
+test.describe("should match image snapshot", () => {
```

Comment:
Maybe use similar layout to property grid e2e tests?
```typescript
test.describe("tree widget" () => {
  const testCases = () => {
    // ...
  }

  test.describe("default", () => {
    testCases();
  });

  test.describe("enlarged", () => {
    test.beforeEach(() => {
      // toggle expanded layout
    });
    testCases();
  });
})
```

Reply:
Changed.